packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -140,6 +142,22 @@ function removeNotTableDefaultStyles(element: HTMLTableElement) {
     });
 }
 
+/**
+ * Verifies recursively if a node and its parents have any siblings
+ * Ignoring the children of contentDiv and returning true if any parent is LI

```

Comment:
Why we have special handling for LI here?

Reply:
lists and its items are always inside a single block (meaning it's gonna find siblings) and would break the logic for clear formatting otherwise as the `<li>` items should be removed

Reply:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/44042957/203359930-bc61bdbd-366b-470a-ad0a-e83ce6df33d7.png">